src/OAuth2.php

```diff
@@ -1072,6 +1071,25 @@ private function coerceUri($uri)
     }
   }
 
+  private function jwtDecode($idToken, $publicKey, $allowedAlgs)
+  {
+    if (class_exists('Firebase\JWT\JWT')) {
+      return \Firebase\JWT\JWT::decode($idToken, $publicKey, $allowedAlgs);
+    }
+
+    return \JWT::decode($idToken, $publicKey, $allowedAlgs);
+  }
+
+  private function jwtEncode($assertion, $signingKey, $signingAlgorithm)
+  {
+    if (class_exists('Firebase\JWT\JWT')) {
+      return \Firebase\JWT\JWT::encode($assertion, $signingKey,
+                       $signingAlgorithm);
+    }
+
+    return \JWT::encode($assertion, $signingKey, $signingAlgorithm);
```

Comment:
I could surround `\JWT` with a `class_exists` as well, and throw an exception saying "you must install either 2.0 or 3.0 of firebase/php-jwt to call these functions". That would be a more obvious error to the user, anyway.